go/private/common.bzl

```diff
@@ -103,3 +103,30 @@ def split_srcs(srcs):
 
 def join_srcs(source):
   return depset() + source.go + source.headers + source.asm + source.c
+
+
+def go_importpath(ctx):
```

Comment:
Sync with master. This is missing the new `library` inheritance code.

Reply:
Damn you and your getting a quick cl in between upload and review :)